README.md

```diff
@@ -18,6 +18,9 @@
 
 ---
 
+__⚠️ Attention ⚠️__: the container name `voxpupuli/container-puppetdb` will be deprecated in 2025-02 and might be removed in the future. Please use `voxpupuli/puppetdb` instead.
```

Comment:
maybe also announce that in a blogpost + mailinglist?

Reply:
blogpost: https://voxpupuli.org/blog/2024/08/02/new-container-names/
mailing list: have no idea how to do that, i'm not so the mail guy